src/write.c

```diff
@@ -3142,6 +3143,11 @@ avifResult avifEncoderFinish(avifEncoder * encoder, avifRWData * output)
             AVIF_CHECKERR(avifSequenceHeaderParse(&sequenceHeader, (const avifROData *)&firstSample->data, codecType),
                           avifGetErrorForItemCategory(item->itemCategory));
             item->av1C = sequenceHeader.av1C;
+            // The MA1B brand: The AV1 profile shall be the Main Profile and the level shall be 5.1
+            // or lower.
+            if (item->av1C.seqProfile != 0 || item->av1C.seqLevelIdx0 > 13) {
+                isMA1B = AVIF_FALSE;
+            }
```

Comment:
Yannis: I am not that familiar with this code. It is important that this for loop (which starts at line 3138) is executed for both image items and image sequences, and that we enter the `if (item->encodeOutput->samples.count > 0)` block (which starts at line 3140) at least once. Otherwise we will incorrectly report that `isMA1B` is true (the initial value). Please check this. Thanks!

Reply:
Here is my understanding:
- A track exists in `avifEncoder` only if the corresponding item is present too, likely with the first frame of the sequence being used as the image item data.
- There is no change of profile/level within a single sequence.

Assuming this is correct, the for loop and the `MA1B` check look good.